src/main.go

```diff
@@ -56,6 +56,24 @@ func getCname(path string) (string, string, error) {
 	return cname, version, nil
 }
 
+func checkEFI(expected_loader_entry string) error {
+	_, err := os.Stat("sys/firmware/efi")
```

Comment:
is this missing a leading slash or is that correct?

Reply:
oh, yes, that's missing, good catch

Reply:
fixed